src/components/ShareTrajectoryModal/index.tsx

```diff
@@ -0,0 +1,183 @@
+                      </h2>
+                      <div className={styles.bodyText}>
+                          <p>
+                              To generate a shareable link, please save the file
+                              in the public cloud using Dropbox, Google Drive,
+                              or Amazon S3 and load the model into Simularium
+                              via URL.
+                              <span className={styles.blueText}>
+                                  {" "}
+                                  Learn more{" "}
+                              </span>
+                          </p>
+                      </div>
+                      <Divider className={styles.divider}></Divider>
+                  </div>
+              ),
+              footer: (
+                  <Button
+                      className={styles.okButton}
+                      type="default"
+                      onClick={closeModal}
+                  >
+                      Ok
+                  </Button>
+              ),
+          }
+        : // if the trajectory is a networked file
+          {
+              content: (
+                  <div className={styles.content}>
+                      <div>
+                          <Input
+                              className={styles.urlInput}
+                              value={url}
+                              disabled
+                          />
+                          <Button type="primary" onClick={copyToClipboard}>
+                              Copy {Link}
+                          </Button>
+                      </div>
+                      <div className={styles.secondLine}>
+                          {" "}
+                          <Checkbox onChange={handleAllowUserInput}></Checkbox>
+                          <p>Start at</p>
+                          <Input
+                              className={styles.numberInput}
+                              disabled={allowTimeInput}
+                              defaultValue={currentTime}
+                              onChange={handleUserInput}
+                          />
+                          <p>
+                              {" "}
+                              /{displayTimes.roundedLastFrameTime}{" "}
+                              {timeUnits ? timeUnits.name : null}{" "}
+                          </p>
+                      </div>
+                      <Divider className={styles.divider}></Divider>
+                  </div>
+              ),
+              footer: (
+                  <Button type="default" onClick={closeModal}>
+                      Close
+                  </Button>
+              ),
+          };
+
+    return (
+        <div className={styles.container}>
+            <CustomModal
+                className={styles.uploadModal}
+                title="Share Trajectory"
+                width={isLocalFile ? 611 : 550}
```

Comment:
why is this happening? where do these numbers come from?

Reply:
they come from the UX mock up!